playbooks/UPLOAD_FILE_TO_CLOUD_STORAGE.md

```diff
@@ -0,0 +1,53 @@
+# cloud.gcp_ops.upload_file
+
```

Comment:
To have isolation between the different playbooks that we will have, I guess it would be better to create a folder inside playbooks only for "upload file to cloud storage"